src/SDKs/EventGrid/management/Management.EventGrid/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure EventGrid Management Library")]
 [assembly: AssemblyDescription("Provides developers with a library to create and manage all Azure EventGrid resources.")]
 
-[assembly: AssemblyVersion("3.0.0.0")]
-[assembly: AssemblyFileVersion("3.0.0.0")]
+[assembly: AssemblyVersion("3.0.1.0")]
```

Comment:
Please do not modify `AssemblyVersion` here

Reply:
Fixed

Reply:
`AssemblyFileVersion` must be bumped